suprise_decorator.py

```diff
@@ -0,0 +1,15 @@
+from functools import wraps
+
+
+def suprise_decorator(func):
+    @wraps(func)
+    def function_wrapper(*args, **kargs):
+        print( 'suprise!')
+    return function_wrapper
+
+@suprise_decorator
+def f():
+    for i in range(900):
+        print(i)
```

Comment:
Variable names - https://workat.tech/machine-coding/tutorial/writing-meaningful-variable-names-clean-code-za4m83tiesy0 - variable names, abbreviation reference is missing

Rename 'i' and 'f'